docker/WORKSPACE.docs

```diff
@@ -44,6 +44,11 @@ new_local_repository(
     build_file = "@//third_party/cublas:BUILD",
 )
 
+local_repository(
```

Comment:
Not sure this will work in all cases. If its not installed bazel will not launch

Reply:
removed it.